lib/cluster/cpg.c

```diff
@@ -252,13 +252,10 @@ crm_cs_flush(gpointer data)
     }
 
     queue_len -= sent;
-    if ((sent > 1) || (cs_message_queue != NULL)) {
-        crm_info("Sent %u CPG messages  (%d remaining): %s (%d)",
-                 sent, queue_len, pcmk__cs_err_str(rc), (int) rc);
-    } else {
-        crm_trace("Sent %u CPG messages  (%d remaining): %s (%d)",
-                  sent, queue_len, pcmk__cs_err_str(rc), (int) rc);
-    }
+    do_crm_log((queue_len > 5)? LOG_INFO : LOG_TRACE,
```

Comment:
I see at least one other place where we use 5 regarding queue_len.  Should this be a constant somewhere?

Reply:
or LOG_PRIO_TRACE or something ...

Reply:
They're only marginally related; this is the CPG message queue and the other use (presumably delay_next_flush()) is the IPC message queue. This is purely a log message, whereas the other affects the sending of messages.

The use here isn't so much a magic number as a ballpark figure for when to complain more loudly.